test/Microsoft.AspNet.Mvc.FunctionalTests/TestHelper.cs

```diff
@@ -131,6 +131,26 @@ private static IServiceProvider BuildFallbackServiceProvider(IEnumerable<IServic
             return sc.BuildServiceProvider();
         }
 
+        public static string NormalizeExpectedUrl(this string url)
```

Comment:
Is there no way to get a utility function from routing to do this work? Seems bad/error prone to be doing all this work for an expected result.